src/client.ts

```diff
@@ -112,9 +121,12 @@ export default class Client extends EventEmitter {
         rid: this.rid,
         uid: this.uid,
       });
-      if (this.local) {
-        this.local.unpublish();
+      for (const localStream of this.localStreams) {
+        if (localStream.mid) {
+          await localStream.unpublish()
```

Comment:
can we parallelize this with `Promise.all`?

Reply:
Done ! :-)